src/main/java/gov/usds/case_issues/services/CaseDetailsService.java

```diff
@@ -69,8 +72,18 @@ public TroubleCase findCaseByTags(String caseManagementSystemTag, String receipt
 	public CaseDetails findCaseDetails(String caseManagementSystemTag, String receiptNumber) {
 		TroubleCase mainCase = findCaseByTags(caseManagementSystemTag, receiptNumber);
 		Collection<CaseIssueSummary> issues = _issueRepo.findAllByIssueCaseOrderByIssueCreated(mainCase);
-		Collection<CaseSnoozeSummary> snoozes = _snoozeRepo.findAllBySnoozeCaseOrderBySnoozeStartAsc(mainCase);
-		List<NoteSummary> notes = _attachmentService.findNotesForCase(mainCase).stream().map(NoteSummary::new).collect(Collectors.toList());
+		List<CaseSnoozeSummary> snoozes = _snoozeRepo.findAllBySnoozeCaseOrderBySnoozeStartAsc(mainCase).stream()
+													.map(row -> new CaseSnoozeSummary(
+														row,
+														_userRepo.findByUserId(row.getCreatedBy())
+													))
+													.collect(Collectors.toList());
+		List<NoteSummary> notes = _attachmentService.findNotesForCase(mainCase).stream()
```

Comment:
We should make sure this takes maximal advantage of either automatic or explicit caches.